src/util/system.cpp

```diff
@@ -356,6 +356,13 @@ static bool InterpretNegatedOption(std::string& key, std::string& val)
     return false;
 }
 
+// Section names to be recognized in the config file.
+const std::set<std::string> ArgsManager::m_available_sections{
```

Comment:
IMO this doesn't have to be exposed in the class and currently it's only necessary in this unit. Suggestion: `static const std::set<std::string> G_CONFIG_SECTIONS{...}` and remove from header.

Reply:
I agree with your opinion. I will fix it tomorrow.

Reply:
@promag fixed and move it just before the place to refer it.